onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -210,6 +211,8 @@ def retrieve(self, request, *args, **kwargs):
         """ Get user profile from cache or db """
         username = kwargs.get('user')
         cached_user = cache.get(f'{USER_PROFILE_PREFIX}{username}')
```

Comment:
```suggestion
        cached_user = cache.get(f'{USER_PROFILE_PREFIX}{username}') or cache.get(f'{ORG_PROFILE_CACHE}{username}')
```

Reply:
https://github.com/onaio/onadata/pull/1890/files#diff-c857800a61f0303b9833f6aa94d239e1R220 Do we need to also set the Org Profile cache here? Could we just have this cached using a single prefix instead i.e `ACCOUNT_PROFILE_PREFIX` ?